### PR TITLE
Improve Docker Images

### PR DESCRIPTION
**Pull request by Malaber**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .git
-test/**
-spec/**
 tmp/**
```
Comment (Thor77) [inline .dockerignore:2]: Why did you remove those? Running `rails spec` locally will probably still create them?

**Review by Thor77 — APPROVED** [review]

File: .dockerignore (hunk 1 above)
Comment (Malaber) [reply .dockerignore:2]: I removed them in order to include them in the Docker Image.

As mentioned in 2ba17a02a0640536d73eed3995a33aaf55df80f9 I would have liked to only include them in the test image, but that doesn't really work, there is also a issue here with the same problem https://github.com/moby/moby/issues/15771
